src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -608,61 +608,61 @@ windowTitle = Window
-windowToolbarLookandfeel = Look and Feel:
-windowToolbarPleaserestart = Please restart logisim.
+windowToolbarLookandfeel = Look and feel:
+windowToolbarPleaserestart = Please restart Logisim.
 windowToolbarPreview = Preview
 windowToolbarReset = Reset window layout to Logisim's defaults
 windowToolbarZoomfactor = Zoom factor
-windowCanvasBgColor = Canvas Background Color:
-windowComponentColor = Component Color:
-windowGridBgColor = Grid Background Color:
-windowGridDotColor = Grid Dot Color:
-windowGridZoomedDotColor = Grid Zoomed Dot Color:
+windowCanvasBgColor = Canvas background color:
+windowComponentColor = Component color:
+windowGridBgColor = Grid background color:
+windowGridDotColor = Grid dot color:
+windowGridZoomedDotColor = Grid zoomed dot color:
 windowGridColorsReset = Reset grid colors to Logisim's defaults
 windowSetAutoScaleFactor = Set zoom factor to Logisim's default
 windowCanvasLocation = Main canvas location:
 #
 # prefs/HotkeyOptions.java
 #
-hotkeyOptMenuKeyHeader = Menu Shortcuts That Require the %s Key:
-hotkeyOptNormalKeyHeader = Context Dependent Hotkeys That Do Not Require the %s Key:
-hotkeyOptOrientDesc = Component's Orientation Shortcuts
+hotkeyOptMenuKeyHeader = Menu shortcuts that require the %s key:
+hotkeyOptNormalKeyHeader = Context dependent hotkeys that do not require the %s key:
+hotkeyOptOrientDesc = Component's orientation shortcuts
 hotkeyOptHelp = To set the key bindings, click the bordered area right-side and press your expected keys in the keypad. Click the check mark to save, or discard by clicking the cross mark.
-hotkeyOptTitle = Hotkey Settings
-hotkeyOptResetBtn = Reset to Default
+hotkeyOptTitle = Hotkey settings
+hotkeyOptResetBtn = Reset to defaults
 hotkeySimAutoPropagate = Auto-Propagate
-hotkeySimReset = Reset Simulation
-hotkeySimStep = Single-Step Propagation
-hotkeySimTickHalf = Manual Tick Half Cycle
-hotkeySimTickFull = Manual Tick Full Cycle
-hotkeySimTickEnabled = Auto-Tick Enabled
-hotkeyEditUndo = Edit Undo
-hotkeyEditRedo = Edit Redo
-hotkeyWindowClose = Close the Window
-hotkeyWindowMinimize = Minimize the Window
-hotkeyFileExport = Export File
-hotkeyFilePrint = Print File
-hotkeyErrMeta = Must contain the %s key.
-hotkeyErrConflict = This key conflicts with: %s.
-hotkeyDirNorth = North
-hotkeyDirSouth = South
-hotkeyDirEast = East
-hotkeyDirWest = West
+hotkeySimReset = Reset simulation
+hotkeySimStep = Single-step propagation
+hotkeySimTickHalf = Manual tick half cycle
+hotkeySimTickFull = Manual tick full cycle
+hotkeySimTickEnabled = Auto-tick enabled
+hotkeyEditUndo = Edit undo
+hotkeyEditRedo = Edit redo
+hotkeyWindowClose = Close the window
+hotkeyWindowMinimize = Minimize the window
+hotkeyFileExport = Export file
+hotkeyFilePrint = Print file
+hotkeyErrMeta = Must contain the %s key
+hotkeyErrConflict = This key conflicts with: %s
+hotkeyDirNorth = ↑ North
+hotkeyDirSouth = ↓ South
+hotkeyDirEast = → East
+hotkeyDirWest = ← West
```

Comment:
I'm not sure if adding the arrows is beneficial.
```
        ↑ North
← West  ↓ South  → East
```
is IMO not better than
```
      North
West  South  East
```
If we decide that we like the arrows, we should consider moving `→` to the right of `East` (`East →`)